tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -30,8 +30,8 @@ type Stresser interface {
 	Stress() error
 	// Cancel cancels the stress test on the etcd cluster
 	Cancel()
-	// Report reports the success and failure of the stress test
-	Report() (success int, failure int)
+	// Report reports the number of entries created and deleted during the stress test
+	Report() int64
```

Comment:
Is `ModifiedKeys()` too specific? i was thinking about use generic term in the interface. But i have no objection in using a more specific term.